MaxMind.MinFraud/WebServiceClient.cs

```diff
@@ -254,7 +255,7 @@ private static async Task Handle4xxStatus(HttpResponseMessage response)
             // set Content for the default response.
             var content = response.Content != null ? await response.Content.ReadAsStringAsync().ConfigureAwait(false) : null;
 
-            if (content == null || content.Length == 0)
+            if (string.IsNullOrEmpty(content))
```

Comment:
The warnings make it look like `netstandard2.0` doesn't treat `IsNullOrEmpty` as a null reference check, is that alright?

Reply:
Yeah, I saw that. It seemed fine to me given that the behavior is still the same. I suppose we could suppress the warning or revert this change, but `netstandard2.0` is the next version to be cut.